HW tablels without Lviv

alex3343alex opened the pull request.

alex3343alex committed:
Resolved first HW Lviv and tables

alex3343alex committed:
HW

alex3343alex committed:
Delete workspace.xml

alex3343alex committed:
Delete WebstormProjects.iml

alex3343alex committed:
Delete Project_Default.xml

alex3343alex committed:
Delete modules.xml

BohdanTsiupryk commented on Lviv/css/styles.css:
```diff
@@ -0,0 +1,23 @@
+    font-size: 20px;
+}
+
+.BackIMG{
```
Перегляньте конвенцію по іменах в css

BohdanTsiupryk commented on Lviv/index.html:
```diff
@@ -0,0 +1,44 @@
+<p><strong style="font-size: large"> Львів </strong>&#8212; найбільше місто Західної України,
+    що протягом багатовікової історії було науковим, культурним та національним центром регіону. </p>
+<p>В найдавніші часи &#8212; столиця <span style="letter-spacing: 5px" class="underLine">Галицько - Волинської держави</span>,
+    згодом &#8212; <span style="letter-spacing: 5px" class="spacing; underLine">адміністративний центр Руського<br>
```
коли перелічуєте класи крапка з комою непотрібна

BohdanTsiupryk replied:
використовуєте класс який має властивість letter-spacing і по місцю переписуєте letter-spacing

BohdanTsiupryk commented on Lviv/css/styles.css:
```diff
@@ -0,0 +1,23 @@
+.alignment{
+    text-align:center;
+}
+.size{
```
Не зовсім коректна назва, size чого? який? для чого? кому? де? як? чим?

alex3343alex committed:
1. Переглянуто конвенцію по іменах в css.
2. Позбувся крапки з комою між класами.
3. Використав клас з параметром letter-spacing.
4. Змінив назву size.

alex3343alex committed:
1.Змінив назву color на background_color

alex3343alex committed:
Merge remote-tracking branch 'origin/master'

alex3343alex committed:
1.Змінив назву color на background_color
